src/modules/eventListeners.js

```diff
@@ -39,18 +39,21 @@ todoList.addEventListener('click', (e) => {
   }
 });
 
+// Clear completed button
 clearCompletedBtn.addEventListener('click', () => {
   const toDoList = clearCompleted(toDoTasks);
   updateLocalStorage(toDoList);
   renderToDoList(toDoList);
 });
 
+// Add event listener for double click to edit
 todoList.addEventListener('dblclick', (e) => {
   if (e.target.closest('.todo-list-li-text')) {
     editTask(e, toDoTasks);
   }
 });
 
+// Delete a task on click to the cross button
```

Comment:
- [ ] According to the project requirement, the updated array of items should be stored in local storage but the index is repeating. Please fix this issue.
- [ ] I notice that when I remove the completed list and then add the new one then it shows the list which is removed already. Please fix this issue. 
- [ ] You can solve this issue by updating the local storage after the user does `clear all completed`. You did not update the clear all button on local storage so it is re-rendering again.
### Error
![his](https://user-images.githubusercontent.com/81727988/213864610-c3fd9a81-d9fc-4b34-9458-ab78b1ffbb96.jpg)